stylelint-polaris/index.js

```diff
@@ -4,414 +4,503 @@ const {
   tokens,
 } = require('@shopify/polaris-tokens');
 
-/** @type {import('./plugins/coverage').PrimaryOptions} */
```

Comment:
Can we keep this type named `PrimaryOptions`? This was intentionally aligned with terminology in the Stylelint docs and [Type definitions](https://github.com/stylelint/stylelint/blob/11acf312191e5097692f7428f13977bd1fd12cd4/types/stylelint/index.d.ts#L192).